manuscript.Rmd

```diff
@@ -379,6 +380,29 @@ The mean RTs obtained over training blocks are shown in the left panel of Figure
 Analysing RTs using a 2 (*material*: permuted vs. random) $\times$ 6 (*block number*) ANOVA revealed a significant main effect of *material*, `r out$full_result$material`, with slower responses for random material than for permuted material, and a significant main effect of *block number*, `r out$full_result[["block"]]`, reflecting practice effects. 
 The factors *material* and *block number* did not interact, `r out$full_result[["material_block"]]`, indicating that any knowledge was acquired already during the first block. 
 
+```{r cache = TRUE}
```

Comment:
I think it would be good to add a chunk name here, so we can locate and load the cached objects more easily if desired.